azurerm_function_app/main.tf

```diff
@@ -101,27 +107,27 @@ resource "azurerm_app_service_plan" "azurerm_app_service_plan" {
   }
 }
 
-# STORAGE ACCOUNT
-resource "azurerm_storage_account" "azurerm_functionapp_storage_account" {
-  name                = "${local.azurerm_functionapp_storage_account_name}"
-  resource_group_name = "${var.resource_group_name}"
-  location            = "${var.location}"
+# # STORAGE ACCOUNT
+# resource "azurerm_storage_account" "azurerm_functionapp_storage_account" {
+#   name                = "${local.azurerm_functionapp_storage_account_name}"
+#   resource_group_name = "${var.resource_group_name}"
+#   location            = "${var.location}"
```

Comment:
Comments are for comments only :) No code commented in the repo besides exceptions please